launch_only_docker.sh

```diff
@@ -0,0 +1,3 @@
+#Bind the host machine's urc-2023 root folder to the container's target path.
+#Launch the "trickfireimage" image by giving shell control over to the container
+docker run --mount type=bind,source="$(pwd)",target="/home/trickfire/urc-2023" -it --privileged trickfireimage
```

Comment:
Not sure if it's important, but should the path be "/urc-2023" instead of "/home/trickfire/urc-2023"? "/home/trickfire" might be a local thing. If this functionality works, then disregard this comment.

Reply:
The difference will only be where the urc-2023 folder lives in, and I would rather not have it live in the root directory - so I put it in /home/trickfire/